NitroxClient/GameLogic/Items.cs

```diff
@@ -59,25 +59,36 @@ public void PickedUp(GameObject gameObject, TechType techType)
             packetSender.Send(pickupItem);
         }
 
-        public void Dropped(GameObject gameObject, TechType techType)
+        /// <summary>
+        /// Tracks the object (as dropped) and notifies the server to spawn the item for other players.
+        /// </summary>
+        public void Dropped(GameObject gameObject, TechType techType = default)
         {
+            if (techType == default)
+            {
+                techType = CraftData.GetTechType(gameObject);
+                if (techType == TechType.None)
+                {
+                    Log.Error($"Could not find {nameof(TechType)} for {gameObject.name}");
+                    return;
```

Comment:
Spawning primarily uses class id (below) for loading the prefab. Tech type is normally just a 'nice to have'.  I don't think we should get ourselves into a desync state if we resolve to TechType.None.

Reply:
but it cannot happen ? @Measurity what do you think ?
In any case, it's just moving the CraftData.GetTechType(GameObject) from the exterior of the function to inside the function

Reply:
Removed the return and logging. Will not just pass in whatever TechType is fetched by `CraftData.GetTechType(gameObject)` unless a non-null `TechType` was already given.

Reply:
Thanks Measurity.  @VELD-Dev just for clarity, via `CraftData.GetTechType`, it seems to be a normal possible case that the prefab identifier exists but there is no mapping of class id to tech type (defaults to none).  

![image](https://user-images.githubusercontent.com/3238547/230977733-3226f5a3-b3f9-4261-bf00-fe0876d02822.png)

Reply:
> Thanks Measurity. @VELD-Dev just for clarity, via `CraftData.GetTechType`, it seems to be a normal possible case that the prefab identifier exists (thus a class id) but there is no mapping of class id to tech type (defaults to none).
> 
> ![image](https://user-images.githubusercontent.com/3238547/230977733-3226f5a3-b3f9-4261-bf00-fe0876d02822.png)

Ohhh okay got it, thanks !